src/notifications.js

```diff
@@ -178,9 +179,78 @@ Notifications.push = function (notification, uids, callback) {
 };
 
 function pushToUids(uids, notification, callback) {
-	var oneWeekAgo = Date.now() - 604800000;
-	var unreadKeys = [];
-	var readKeys = [];
+	function sendNotification(uids, callback) {
```

Comment:
Maybe pull out all of the email-related stuff into `src/notifications/email.js`?